pkg/sdn/plugin/node.go

```diff
@@ -70,10 +75,13 @@ func NewNodePlugin(pluginName string, osClient *osclient.Client, kClient *kclien
 		}
 	}
 
+	podChan := make(chan *podRequest)
 	plugin := &OsdnNode{
 		multitenant:        IsOpenShiftMultitenantNetworkPlugin(pluginName),
 		kClient:            kClient,
 		osClient:           osClient,
+		cniServer:          NewCniServer(podChan),
+		podRequestChan:     podChan,
```

Comment:
Then we couldn't pass that channel to NewCniServer() though.  It needs to be the same channel, hence why it got broken out.